### PR TITLE
Fixing color order should match node order

### PR DESCRIPTION
**Pull request by jsignell**
Closes: #223 

Using example from issue:

Screen Shot 2019-08-08 at 11 30 06 AM

Not sure how to test.

### CODE REVIEW

File: hvplot/networkx.py [hunk 1]
```diff
@@ -75,7 +75,7 @@ def _from_networkx(G, positions, nodes=None, cls=Graph, **kwargs):
     node_columns = defaultdict(list)
 
     # Unpack node positions
-    for idx, pos in sorted(positions.items()):
+    for idx, pos in positions.items():
```
Comment (philippjfr) [inline hvplot/networkx.py]: Wonder why I sorted here in the first place.

Comment (jsignell) [reply hvplot/networkx.py]: Hard to tell why since there aren't any tests :(

**Comment by philippjfr** [comment]
>Not sure how to test.

Check the order of this maybe?

```
plot.nodes.dimension_values(2)
```

File: hvplot/networkx.py (hunk 1 above)
Comment (philippjfr) [reply hvplot/networkx.py]: 😞

**Pull request merged**